native/rust/rspdk/src/main/rust/Cargo.toml

```diff
@@ -16,7 +16,7 @@ license = "Apache-2.0"
 authors = [ "Alluxio Developers" ]
 
 [lib]
-crate-type = [ "cdylib" ]
+crate-type = [ "cdylib", "rlib" ]
```

Comment:
`rlib` was needed to allow generating the rust library for consumption when compiling the binary.